packages/nextra-theme-docs/src/components/footer.tsx

```diff
@@ -8,7 +8,7 @@ import { renderComponent } from '../utils'
 export function Footer({ menu }: { menu?: boolean }): ReactElement {
   const config = useConfig()
   return (
-    <footer className="nx-bg-gray-100 nx-pb-[env(safe-area-inset-bottom)] dark:nx-bg-neutral-900">
+    <footer className="nx-bg-gray-100 nx-pb-[env(safe-area-inset-bottom)] dark:nx-bg-neutral-900 print:nx-bg-transparent">
```

Comment:
I guess we can hide the footer while printing, also maybe we should hide the breadcrumb also?

And I like the idea of using `print:*` classes!

Reply:
In my opinion, I prefer save the footer and the breadcrumb. The footer tells the author and license information of the page. And the breadcrumb can be seen as the page header showing the current chapter. Plus, they are useful when the page is exported to PDF. Maybe we can simply make the footer take less area when being printed?